packages/gasket-react-intl/src/with-locale-required.js

```diff
@@ -10,87 +11,114 @@ const { defaultLocale, defaultPath } = manifest;
  * Sets up and attaches the getInitialProps static method which preloads locale
  * files during SSR for Next.js pages. For browser routing, the locale files
  * will be fetched as normal.
- *
- * @param {React.ComponentType} Wrapper - The HOC
- * @param {LocalePathPart} localePathPart - Path containing locale files
+ * @type {import('./index').attachGetInitialProps}
  */
 function attachGetInitialProps(Wrapper, localePathPart) {
   const { WrappedComponent } = Wrapper;
 
+  /** @type {import('./index').attachedGetInitialProps} */
+  // @ts-ignore - attaches getInitialProps to Wrapper
   Wrapper.getInitialProps = async (ctx) => {
     const { res } = ctx;
     let localesProps;
 
     let resolvedLocalePathPart;
     if (typeof localePathPart === 'function') {
-      // While this can be resolved by serverLoadData, we will do it here
-      // and return in it in props to avoid having to re-resolve during hydrate under different context
-      resolvedLocalePathPart = localeUtils.resolveLocalePathPart(localePathPart, ctx);
+      // While this can be resolved by serverLoadData, we will do it here and
+      // return in it in props to avoid having to re-resolve during hydrate
+      // under different context
+      // @ts-ignore - temporary until we fix types in @gasket/helper-intl
+      resolvedLocalePathPart = localeUtils.resolveLocalePathPart(
+        localePathPart,
+        ctx
+      );
     }
 
     if (res && res.locals && res.locals.gasketData) {
       const { locale = defaultLocale } = res.locals.gasketData.intl || {};
       const localesParentDir = require('path').dirname(res.locals.localesDir);
-      localesProps = localeUtils.serverLoadData(resolvedLocalePathPart ?? localePathPart, locale, localesParentDir);
+      // @ts-ignore - temporary until we fix types in @gasket/helper-intl
```

Comment:
Were you able to fix this infact?

Reply:
Looks like removing this ignore and the ignore on line 30 works

Reply:
Awesome. Will update!